src/components/CompareResults/TableHeader.tsx

```diff
@@ -72,19 +77,29 @@ function FilterableColumn({
             theme.palette.mode == 'light'
               ? Colors.Background200
               : Colors.Background200Dark,
-          borderRadius: '4px',
-          fontSize: 'inherit',
+          borderRadius: 0.5,
+          padding: '6px 12px',
         })}
       >
-        {name}
         <FilterListIcon
           fontSize='small'
           color={hasFilteredValues ? 'primary' : 'inherit'}
-          sx={{ marginInlineStart: 1 }}
+          sx={{ marginInlineStart: 1, marginRight: 0.5, marginLeft: 0 }}
           titleAccess={
             hasFilteredValues ? 'Some filters are active' : 'No active filters'
           }
         />
+        {name}
+        <Box
+          sx={{
+            paddingLeft: 0.5,
+            color: themeMode == 'light' ? Colors.LinkText : Colors.LinkTextDark,
+            fontWeight: 800,
```

Comment:
I'd remove the boldness for now. In Dasha's design the text is bold as well, but then all headers should be bold, and I think this is out of scope for this PR. When we'll add the boldness to all headers we can also add it here.